aws/vpc/main.tf

```diff
@@ -14,6 +14,7 @@ module "vpc" {
   public_subnet_tags = {
     "kubernetes.io/cluster/${var.cluster_name}" = "shared"
     "kubernetes.io/role/elb"                    = "1"
+    "tier" = "public"
```

Comment:
Are you sure about it?

Reply:
nope.. was a guess

a8b41b2e33894b8e8a0f31c796bee510e62b9291